.github/ISSUE_TEMPLATE.md

```diff
@@ -1,58 +0,0 @@
-<!---
```

Comment:
+1 to having a separate, targeted template for each type of issue

Reply:
There's also a reference from Ansibot to this file, which I have a small fix for (pointing to the directory instead). But yes, if it is still being used somewhere, we have to keep it around. I'll fix it up too.

Reply:
Fixed.